langchain4j-core/src/main/java/dev/langchain4j/data/message/ContentType.java

```diff
@@ -12,7 +12,19 @@ public enum ContentType {
     /**
      * Image content.
      */
-    IMAGE(ImageContent.class);
+    IMAGE(ImageContent.class),
+    /**
+     * Audio content.
+     */
+    AUDIO(AudioContent.class),
```

Comment:
Please register adapters for `AudioContent` and `VideoContent` in `GsonChatMessageAdapter`:

```java
    private static final Gson GSON = new GsonBuilder()
            .registerTypeAdapter(Content.class, new GsonContentAdapter())
            .registerTypeAdapter(TextContent.class, new GsonContentAdapter())
            .registerTypeAdapter(ImageContent.class, new GsonContentAdapter())
            .create();
```

Would be also great to update `ChatMessageSerializerTest.should_serialize_and_deserialize_chat_message` with `AudioContent` and `VideoContent`.